### ISSUE TITLE
oAuth returns access_denied

### ISSUE THREAD

**Issue by ScriptPup** [Patreon/patreon-js#10]
Hi there. I know this is going to come off weird since it's not really an issue with your library... Well, kinda. I can't really figure out how I'm supposed to use this module from the documentation, but that's beside the point.

I have built out my website for the oAuth requests and whatnot as specified by the patreon documentation but always get an access_denied error as step 3 ([documentation](https://www.patreon.com/platform/documentation/oauth)). I can't find any real support channels for this, but I figured you've been able to get it to work (or a module wouldn't exist). maybe you'd be willing to take a look at my question on SO? http://stackoverflow.com/questions/40391941/nodejs-patreon-api-account-link

If not I get it, but it seems like no one really posts anything about this... I'll fall back on the classic plea, "Help me Obi-Wan, you're my only hope."

**Comment by 21echoes** [comment]
Hey @ScriptPup ! Would love to help you out both with debugging your linked code not using our library, and get you set up using this actual library. The email linked at the top of the [documentation overview](https://www.patreon.com/platform/documentation/overview) will probably be a more fruitful channel for fixing your existing code, as knowing your patreon user will help us trace down precisely what's happening to you.

**Comment by ScriptPup** [comment]
Thank you very much for your response @21echos ! I actually found a support link (didn't see the email address somehow) and submitted a ticket. About 10 minutes after that I went and re-did all of my previous steps from scratch and it's now working for some reason. I'm chalking it up to an invisible character or something.

Thank you again for responding to me when closing the ticket would have been much easier. I should be good and am happy to call the issue resolved. I'll post my final configuration on SO later tonight so others can see how it's working.
